<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionAccountWidget</class>
 <widget class="QWidget" name="FunctionAccountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>130</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>130</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>130</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_normal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: bold 16px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>账户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_asset">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>130</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>资产</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_miner">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>97</y>
     <width>130</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>挖矿</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bonus">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>134</y>
     <width>130</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分红</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
